src/world.cpp

```diff
@@ -52,9 +52,6 @@ bool World::init(vec2 screen)
 	glfwWindowHint(GLFW_CONTEXT_VERSION_MINOR, 3);
 	glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE);
 	glfwWindowHint(GLFW_OPENGL_DEBUG_CONTEXT, 1);
-#if __APPLE__
-	glfwWindowHint(GLFW_OPENGL_FORWARD_COMPAT, GL_TRUE);
-#endif

```

Comment:
can you actually add this back in? I know we're not actively trying to support Mac now, but it's nice to at least have a partial version working so I can test stuff in class, and it won't compile at all without this

Reply:
Will do.